gui/wxpython/gmodeler/model.py

```diff
@@ -2256,7 +2256,7 @@ def _processConditions(self):
             pos, size = self._getDim(node)
             text = self._filterValue(self._getNodeText(node, "condition")).strip()
             aid = {"if": [], "else": []}
-            for b in aid.keys():
+            for b in aid.keys():  # noqa: PLC0206
```

Comment:
I was conservative here. I'm not sure using `key, value` would behave when appending to `value` during iteration.